### PR TITLE
adding decision tree for outreach intervention

### PR DESCRIPTION
**Pull request by catherineojohnson**
Updated to add decision tree and algorithm for outreach intervention

### CODE REVIEW

**Review by ndecleene — APPROVED** [review]

**Comment by yongqx** [comment]
It's kind of too many file changes in one PR. I suggest divide the whole piece into different sections and add labels in the future.

File: docs/source/concept_models/vivarium_us_cvd/.smbdeleteAAA164eb10d [hunk 1]
```diff
@@ -0,0 +1,338 @@
+.. _us_cvd_concept_model:
```
Comment (mattkappel) [inline docs/source/concept_models/vivarium_us_cvd/.smbdeleteAAA164eb10d:1]: I assume you don't want this file in your PR. You'll need to remove it (`git rm`) locally, commit, and push to do that. Or I think there's also a way to delete this file in the Github web UI.

File: docs/source/intervention_models/crm_mgmt/index.rst [hunk 2]
```diff
@@ -7,12 +7,12 @@ Cardiometabolic Risk Management Intervention Model
 This model includes behavioral and pharmaceutical interventions aimed
 at: 1) improving blood pressure and LDL-cholesterol control, 2)
 increasing exercise, 3) decreasing BMI/weight, 4) improving control of
-fasting plasma glucose, and 5) decreasing tobacco use. Intervening to
-change an individual's risk factor exposures will improve outcomes for
-many GBD causes.
+fasting plasma glucose, and 5) decreasing tobacco use. Altering an individual's 
+risk factor exposures is expect to improve outcomes for many cardiometabolic
```
Comment (mattkappel) [inline docs/source/intervention_models/crm_mgmt/index.rst:11]: `s/is expect/is expected/`

**Review by mattkappel — APPROVED** [review]
This looks good and makes sense. We'll need more details around the several places we need to sample from distributions (how, which distribution, etc.) and around how to calculate and use the ASCVD score in this model, but those details can come in another PR.

**Comment by mattkappel** [comment]
Could this PR be merged? It would be helpful to see it in the published docs. I'm happy to help with the conflict resolution bit, too!

**Pull request merged**